src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -122,12 +123,7 @@
 
             public void AddLocal(LocalSymbol local)
             {
-                if (_locals == null)
-                {
-                    _locals = ArrayBuilder<LocalSymbol>.GetInstance();
-                }
-
-                _locals.Add(local);
+                (_locals ??= ArrayBuilder<LocalSymbol>.GetInstance()).Add(local);
```

Comment:
>(_locals ??= ArrayBuilder<LocalSymbol>.GetInstance()).Add(local); [](http://example.com/codeflow?start=16&length=65)

Is this a pure refactoring, i.e. no behavior changes are intended? If so, please revert changes like that. We prefer to not mix pure refactorings with functional changes in the same PR, unless they are really necessary, like extracting common helper to use in new code.